script.js

```diff
@@ -1,4 +1,69 @@
+var currentGameMode = "waiting for user name";
+var userName = "";
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var myOutputValue = "";
+  var computerChosenSecretWord = chooseSecretWord();
+  // if the user didn't enter a name
+  if (isNaN(Number(input)) == true) {
+    myOutputValue = "sorry please enter a name";
+  } else {
+    //otherwise play the game
+    userName = input;
+    currentGameMode = "scissors paper stone game";
+    myOutputValue = `Hello ${input}!`;
+  }
+  if (currentGameMode == "scissors paper stone game") {
```

Comment:
Consider having a shorter name to compare to currenGameMode. An example would just be 'in-game' this is to reduce errors that might occur when typing long strings